utils/data_loader.py

```diff
@@ -0,0 +1,56 @@
+import torch
```

Comment:
This file can be a notebook which may live in `split_creator` directory and may have same name as `data_config_{identifier}` i.e. `split_creator/data_config_{identifier}.ipynb`. You, the designer runs it once and it saves the data at some path which is then specified in `data_config_{identifier}.toml`